recipes/r-intrinsic/meta.yaml

```
@@ -0,0 +1,109 @@
+{% set version = '0.2.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-intrinsic
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/intRinsic_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/intRinsic/intRinsic_{{ version }}.tar.gz
+  sha256: efc2468da25dbc24aaceabd937f9dc44ffb5dcd751524a3622ceb588fd0d7a88
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - _openmp_mutex                # [linux]
```

see https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp
```suggestion
    - llvm-openmp  # [osx]
    - libgomp      # [linux]
```